src/main/java/hudson/plugins/git/GitTool.java

```diff
@@ -137,7 +138,7 @@ public static void onLoaded() {
     }
 
 
-    @Extension
+    @Extension @Symbol("git")
```

Comment:
There is the jgit one here already. I'd tend to go with git for this one
since it's the default and use distinctive names for the others, but I am
persuadable. =)